File: LeetCode/Easy/387-first-unique-character-in-a-string/try-2.c
/*
 * Problem #387: First Unique Character in a String
 * Difficulty: Easy
 * Submission: Try 2
 * status: Accepted
 * Language: c
 * Date: 7/4/2026, 6:35:48 PM
 * Link: https://leetcode.com/problems/first-unique-character-in-a-string/
 */

int firstUniqChar(char* s) {
    int hash[26]={0};
    for(int i=0;s[i]!=0;i++){
        hash[s[i]-'a']++;
    }
    int ans=-1;
    for(int i=0;s[i]!=0;i++){
        if(hash[s[i]-'a']==1){
            ans=i;
            break;
        }
    }
    return ans;
    }

File: LeetCode/Easy/1678-goal-parser-interpretation/try-2.py
/*
 * Problem #1678: Goal Parser Interpretation
 * Difficulty: Easy
 * Submission: Try 2
 * status: Accepted
 * Language: python3
 * Date: 7/9/2026, 10:31:59 AM
 * Link: https://leetcode.com/problems/goal-parser-interpretation/
 */

class Solution:
    def interpret(self, command: str) -> str:
        return command.replace("()", "o").replace("(al)", "al")
